### PR TITLE
Bundle benchmarking scripts together

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

**Review by erikd — APPROVED** [review]

File: scripts/benchmarking/README.md [hunk 1]
```diff
@@ -52,18 +52,16 @@
 
 ### Index
 
-- <span><b>chairman.sh</b></span> <br/>  Run the Chairman consensus checker against a cluster started
-                     by `shelley-testnet*.sh` family of scripts
```
Comment (newhoggy) [inline scripts/benchmarking/README.md:56]: What's the reason for deleting this line?

Comment (Jimbo4350) [reply scripts/benchmarking/README.md:56]: The chairman script being referred to was very outdated. I want to set a clear boundary between what scripts the benchmarking team use (and therefore should maintain) and what the node team use.

**Review by newhoggy — APPROVED** [review]

**Review by intricate — APPROVED** [review]

**Review by deepfire — APPROVED** [review]
LGTM

**Pull request merged**